flow-typed/Claim.js

```diff
@@ -191,32 +191,3 @@ declare type ChannelPublishParams = {
   tags?: Array<string>,
   languages?: Array<string>,
```

Comment:
- Create a public playlist called "throw"
- After confirmed, edit the title to "Don't Throw" and save.
- When publishing the edits:
    - :small_red_triangle:  the `name` becomes `Don't-Throw`  -- shouldn't the name be immutable?
    - :small_red_triangle:  the `title` becomes `Don't-Throw` -- spaces should be allowed here
        - But the API used `Don't Throw` correctly ... display issue?
    - :grey_question:   I can't recall what's the original channel (but pretty sure if wasn't anon), but the final publish ended up making the list "anonymous"